drivers/hmis/app/jobs/hmis/auto_exit_job.rb

```diff
@@ -0,0 +1,80 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Hmis
+  class AutoExitJob < BaseJob
+    def perform
+      Hmis::Hud::Project.all.each do |project|
+        config = Hmis::AutoExitConfig.config_for_project(project)
+        next unless config.present?
+
+        project.enrollments.each do |enrollment|
+          most_recent_contact = if HudUtility2024.residential_project_type_ids.include?(project.project_type)
```

Comment:
Bed Night should only be used as the most recent contact if this is a Night-by-night shelter. Other project types don't record bed-nights.
```suggestion
          most_recent_contact = if project.project_type == 1 # Night-by-night Emergency Shelter
```